sdk/src/types/block/output/mod.rs

```diff
@@ -323,9 +323,12 @@ impl Output {
             (None, Some(Self::Nft(next_state))) => NftOutput::creation(next_state, context),
 
             // Transitions.
-            (Some(Self::Alias(current_state)), Some(Self::Alias(next_state))) => {
-                AliasOutput::transition(current_state, next_state, context)
-            }
+            (Some(Self::Alias(current_state)), Some(Self::Alias(next_state))) => AliasOutput::transition(
+                current_state,
+                next_state,
+                &context.input_chains,
+                context.essence.outputs(),
+            ),
```

Comment:
You don't really need to change that? I'd revert

Reply:
Was needed because of the same name, changed the name and reverted